Fix:
configure:1104:1: error: implicit declaration of function 'XtMalloc' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
configure:1870:67: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
configure:2269:3: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
configure:2269:8: error: implicitly declaring library function 'memcmp' with type 'int (const void *, const void *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2001-03-19 00:52:37.000000000 -0600
+++ configure	2021-07-29 10:48:48.000000000 -0500
@@ -1099,6 +1099,7 @@
 cat > conftest.$ac_ext <<EOF
 #line 1101 "configure"
 #include "confdefs.h"
+#include <X11/Intrinsic.h>
 
 int main() {
 ${x_direct_test_function}()
@@ -1863,6 +1864,7 @@
 #line 1864 "configure"
 #include "confdefs.h"
 #include <ctype.h>
+#include <stdlib.h>
 #define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
 #define TOUPPER(c) (ISLOWER(c) ? 'A' + ((c) - 'a') : (c))
 #define XOR(e, f) (((e) && !(f)) || (!(e) && (f)))
@@ -2262,6 +2264,8 @@
   cat > conftest.$ac_ext <<EOF
 #line 2264 "configure"
 #include "confdefs.h"
+#include <stdlib.h>
+#include <string.h>
 
 main()
 {
